src/ZendCodingStandard/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copied from Squiz_Sniffs_Commenting_VariableCommentSniff
+ * with modifications:
+ *  - use suggested types from ZendCodingStandard\CodingStandard
+ *  - skip normal comments if there is one blank line after the comment
+ */
+
+namespace ZendCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+use ZendCodingStandard\CodingStandard;
+
+class VariableCommentSniff extends AbstractVariableSniff
+{
+    /**
+     * @inheritDoc
```

Comment:
No. Please see: https://www.phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag

Basically we should use `@inheritDoc` (without `{}`) if we want explicitly document the method without any changes and with `{}` when we are adding some additional information.